fieldservice_sale_stock/readme/INSTALL.md

```diff
@@ -0,0 +1,5 @@
+To install Field Service and have the mapping features, you need to
+install GeoEngine.
+
+Please refer to the installation instructions available at:
+<https://github.com/OCA/geospatial/tree/12.0/base_geoengine>
```

Comment:
I think this is an unrelated instruction that has been there since 12.0 

Can you remove the file from this addon?